src/en/authors-charm-store.md

```diff
@@ -19,7 +19,12 @@ In order to interact with the charm store you will need the latest
 [Ubuntu SSO account](https://login.ubuntu.com/+login), and you must have
 logged in to [Launchpad](https://launchpad.net/+login) at least once.
 
-## Login to the store
+## Log in to the store
+
+It is required that you first log in to the
+[Charmstore](https://jujucharms.com) before you attempt to log in via `charm
+login`. After logging in via your browser, you can use the `charm`
```

Comment:
I think "After" could be considered conjunctive here, which would make this an Oxford comma.